settings/post-process-internal.sql

```diff
@@ -0,0 +1,129 @@
+-- Remove geometry from relation_members table an deduplicate
+DROP TABLE IF EXISTS i_members;
+CREATE TABLE i_members AS
+SELECT DISTINCT ON (member_type, member_osm_id)
+  member_type AS osm_type,
+  member_osm_id AS osm_id,
+  geom
+FROM
+  osm_relation_members
+ORDER BY
+  member_type,
+  member_osm_id
+;
+
+--ALTER TABLE osm_relation_members DROP COLUMN geometry;
+
+
+-- Remove from i_members object alredy fully loaded from nodes or ways 
+DELETE
+FROM
+  i_members
+USING
+  osm_nodes
+WHERE
+  i_members.osm_type = 0 AND
+  i_members.osm_id = osm_nodes.osm_id
+;
+
+DELETE
+FROM
+  i_members
+USING
+  osm_ways
+WHERE
+  i_members.osm_type = 0 AND
```

Comment:
`i_members.osm_type = 1`

Reply:
Yes thank you. I have already see it.
Maybe it's a bit early to review the code. I am refactoring and cleaning it.